tslearn/metrics.py

```diff
@@ -79,7 +79,7 @@ def njit_dtw(s1, s2, mask):
         Second time series
 
     mask : array, shape = (sz1, sz2)
-        Mask. Unconsidered cells must have infinite values. d
+        Mask. Unconsidered cells must have infinite values.
```

Comment:
Just a general comment: but wouldn't it be somewhat more user-friendly/intuitive to accept a binary matrix here and just multiply it (or 1 - matrix) with np.inf ourselves?

Reply:
* From a Python point of view, I would agree: When I think of masks, I usually think of a numpy.array with boolean dtype (with True and False values only).

* From a math point of view, I would disagree: It's quite standard to use infinite values for dynamic programming.

However, it's a Python package so I would agree ;)

Reply:
The boolean numpy array as you suggested is a better alternative than a binary matrix as well I'd say!

Reply:
OK, but I will not change it in this PR (for readability), let's do that afterwards.